viewpost.php

```diff
@@ -0,0 +1,20 @@
+<?php
+include ('config/init.php');
+$post = returnPostInformation($_REQUEST['postID']);
+siteHeader('blog * '.$post['title'].'','blog + inspiration');
+
+echo "
+<div class='postSubHeader'>
+<h2>$post[header]:</h2>
```

Comment:
You may or may not care about this, but from an SEO (search engine optimization) standpoint, you probably want the header of the post to be in the `<h1>` tag for the page. The reason why is a bit complicated, so ask me about it sometime. You can leave things like this for now since it won't impact the actual functionality of the page.